### DISCUSSION TITLE
searching Logs for multiple namespaces

### DISCUSSION THREAD

**Discussion by Yiyito11** [signalfx/splunk-otel-collector-chart#620] [Q&A] [answered]
Hello All.
So I installed otelcollector in AKS  in a namespace "foo" and deployed a sample app on same namespace "foo". After that I´m able to check logs of that sample app on splunk cloud without any issues. 
But if I install same application in namespace "bar", I´m not able to search for those logs in that namespace in splunk cloud. I get 0 results with same query I used before on the ther namespace-
Is there a way in  in values.yml to allow otelcollector search of apps from ALL namespaces? (or the ones that I indicate)
Thank you!

**Comment by atoulme** [comment, marked as answer]
No, we don't currently support multiple installations of the helm chart in different namespaces. See #572 for reference.
